conda_build/cli/main_build.py

```diff
@@ -482,12 +487,23 @@ def parse_args(args: Sequence[str] | None) -> tuple[ArgumentParser, Namespace]:
             "Do not display value of environment variables specified in build.script_env."
         ),
     )
-
+    parser.add_argument(
+        "--package-format",
+        dest="conda_pkg_format",
+        choices=CondaPkgFormat.acceptable(),
+        action=PackageTypeNormalize,
+        default=CondaPkgFormat.normalize(
+            context.conda_build.get("pkg_format", conda_pkg_format_default)
+        ),
+        help=(
+            "Choose which package type(s) are outputted. (Accepted inputs tar.bz2 or 1, .conda or 2)"
+        ),
+    )
     add_parser_channels(parser)
 
     parsed = parser.parse_args(args)
     check_recipe(parsed.recipe)
-    return parser, parsed
+    return parsed
```

Comment:
This seems to break a few tests that assume the parsed args are returned as well

```suggestion
    return parser, parsed
```